### DISCUSSION TITLE
Github Students

### DISCUSSION THREAD

**Discussion by ttegun** [community/community#143632] [GitHub Education]
### Select Topic Area

Question

### Body

I've tried enrolling my GitHub student account, but my school doesn't show up on the map on the student account enrollment page. How do I fix it?

[redacted by moderator]

**Comment by ebndev** [comment]
Hi @ttegun, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `GitHub Education` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction.

In the meantime, please feel encouraged to explore existing discussions in the category and utilize the search in case other users may have run into the same issue!
